Firestore/core/src/firebase/firestore/core/firestore_client.h

```diff
@@ -17,10 +17,6 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_FIRESTORE_CLIENT_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_FIRESTORE_CLIENT_H_
 
```

Comment:
I think this is still true, no? This class `import`s `<Foundation>` and has some Objective-C pointers.